index.html

```diff
@@ -0,0 +1,87 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a class="nav-logo" href="index.html">Mahdi Noori</a>
+        <svg
+          width="18"
+          height="12"
+          viewBox="0 0 18 12"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          class="nav-btn"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
+            fill="#6070FF"
+          />
+        </svg>
+      </nav>
+      <section class="headline container">
+        <h1 class="primary-header">I’m Mahdi. Glad to see you!</h1>
+        <p class="headline-description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <div class="social-media">
+          <h2 class="secondary-header">LET’S CONNECT</h2>
+          <ul class="social-icons">
+            <li>
+              <a href="#"
+                ><img
+                  src="images/social-media/twitter-icon.svg"
+                  alt="Twitter icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/social-media/Linkedin-icon.svg"
+                  alt="Linkdin Logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/social-media/medium-icon.svg"
+                  alt="Medium Logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/social-media/github-icon.svg"
+                  alt="Github Logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/social-media/anglist-icon.svg"
+                  alt="Anglist Logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </header>
```

Comment:
- [ ] Kindly I recommend you to use` header` tag only for as a page header instead of using for all headline section, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Just like this : 
```
<header>
      <nav class="navbar">
        <a class="nav-logo" href="index.html">Mahdi Noori</a>
        <svg
          width="18"
          height="12"
          viewBox="0 0 18 12"
          fill="none"
          xmlns="http://www.w3.org/2000/svg"
          class="nav-btn"
        >
          <path
            fill-rule="evenodd"
            clip-rule="evenodd"
            d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
            fill="#6070FF"
          />
        </svg>
      </nav>
      </header>
```